core/src/main/scala/cats/instances/ordering.scala

```diff
@@ -0,0 +1,16 @@
+package cats
+package instances
+
+import cats.functor.Contravariant
+
+trait OrderingInstances {
+
+  implicit val catsFunctorContravariantForOrdering: Contravariant[Ordering] =
+    new Contravariant[Ordering] {
```

Comment:
can't this be `ContravariantCartesian` as well? Just use left to right ordering?

Reply:
Yup. Initially I was uncertain about adding this but ended up going for it.